ckanext/qdes/access/middleware.py

```diff
@@ -7,18 +7,20 @@
 
 
 def qdes_access_before_request():
+    # We could remove it after CKAN 2.10.5 release
+    # if not toolkit.request.endpoint:
+    #     return
```

Comment:
What is this for?

Reply:
i will remove, when i was starting the project, `toolkit.request` was None for some reason. So that was a hack to make the portal work but now can be removed the commented part